.github/workflows/Global - Integration - AAD.yml

```diff
@@ -63,13 +63,15 @@ jobs:
 
           try
           {
-            $Result = Test-DSCConfiguration -Detailed -Verbose
+            $Result = Test-DSCConfiguration -Detailed -Verbose -ErrorAction Stop
           }
           catch
           {
             throw $_
           }
 
+          Write-Host ""
```

Comment:
Why the empty Write-Host?

Reply:
To leave an empty line between whatever comes out of Test-DSCConfiguration and the results below, at least for me it's easier to look at the results that way otherwise they are stuck together.